selftests/functional/test_basic.py

```diff
@@ -476,6 +477,21 @@ def test_invalid_python(self):
         self.assertIn('1-%s:MyTest.test_my_name -> TestError' % test,
                       result.stdout)
 
+    @unittest.skipIf(process.system("which /bin/read", ignore_status=True),
+                     "/bin/read not available.")
+    def test_read(self):
+        try:
+            cmd = utils_path.find_command("reada")
```

Comment:
Sorry about the typo, I was checking whether it works fine. The reason I'm skipping it from within the test is that I need the value from `find_command`. It does not sounds right to use decorator and then execute the very same command again.